pkgs/applications/editors/brackets/default.nix

```diff
@@ -9,55 +11,56 @@ let
     ];
   };
 in
-stdenv.mkDerivation rec {
-  name = "brackets-${version}";
-  version = "1.8";
-
-  src = fetchurl {
-    url = "https://github.com/adobe/brackets/releases/download/release-${version}/Brackets.Release.${version}.64-bit.deb";
-    sha256 = "0b2k0vv1qwmsg1wckp71yrb86zp8zisskmzzvx9ir19bma9jzr42";
-    name = "${name}.deb";
-  };
+  stdenv.mkDerivation rec {
```

Comment:
@joachifm me neither; as I am in the branch and yours didnt work either; lately it has been doing that for other packages too